src/libraries/System.Private.CoreLib/src/System/Buffers/Binary/BinaryPrimitives.ReadBigEndian.cs

```diff
@@ -75,6 +75,28 @@ public static long ReadInt64BigEndian(ReadOnlySpan<byte> source)
                 MemoryMarshal.Read<long>(source);
         }
 
+        /// <summary>
+        /// Reads an Int128 out of a read-only span of bytes as big endian.
```

Comment:
We have some variation in our XML comments here, but it looks like the official docs evolved for all of these to be more of the form:
```suggestion
        /// Reads a <see cref="Int128"/> from the beginning of a read-only span of bytes, as big endian.
```

Reply:
Did a general fixup for the APIs in the impacted files.